core/src/saros/net/xmpp/contact/AddContactHelper.java

```diff
@@ -0,0 +1,148 @@
+package saros.net.xmpp.contact;
+
+import java.util.Objects;
+import java.util.function.BiPredicate;
+import org.apache.log4j.Logger;
+import org.jivesoftware.smack.Connection;
+import org.jivesoftware.smack.XMPPException;
+import saros.SarosConstants;
+import saros.exceptions.OperationCanceledException;
+import saros.net.util.XMPPUtils;
+import saros.net.xmpp.JID;
+import saros.net.xmpp.XMPPConnectionService;
+
+/**
+ * This class is mainly moved code from Eclipse {@code AddContactWizard} to add a contact on the
```

Comment:
This information seems to be interesting in the context of the review, is this information still informative in the future. I think a class description is more useful.